src/app/[locale]/(app)/analytics/page.tsx

```
@@ -0,0 +1,574 @@
+  }
+
+  return (
+    <Card className='mb-6'>
+      <CardHeader>
+        <CardTitle className='text-lg'>Filtros</CardTitle>
+        <CardDescription>
+          Ajuste os filtros para personalizar a visualização dos dados
+        </CardDescription>
+      </CardHeader>
+      <CardContent>
+        <div className='grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-4'>
```

ESLint

[error] Replace `md:grid-cols-2·lg:grid-cols-4·gap` with `gap-4·md:grid-cols-2·lg:grid-cols`

(prettier/prettier)